codebyte added to Cpp fabs , sin and sinh function. (#2324)
Codecademy/docs: content/cpp/concepts/math-functions/terms/fabs/fabs.md

  // Output: 6.5
}
```

> ### Codebyte Example

Reviewer: ```suggestion
## Codebyte Example
```

> The following code example returns the absolute value with the help of the `fabs()` method.

Reviewer: ```suggestion
The following code example returns the absolute value of a decimal with the `fabs()` method.
```

```codebyte/cpp

> 

Reviewer: remove the extra empty lines at the top and bottom of the code block

#include <iostream>

> #include <cmath> // for fabs

Reviewer: ```suggestion
#include <cmath> // For the fabs method
```
